src/services/api/routes/v1/user.ts

```diff
@@ -62,6 +62,10 @@ router.get('/', async (request: express.Request, response: express.Response): Pr
 		connections: {
 			discord: {
 				id: pnid.connections.discord.id
+			},
+			stripe: {
+				tier_name: pnid.connections.stripe.tier_name,
+				tier_level: pnid.connections.stripe.tier_level
```

Comment:
I don't know what the types are here, but I reckon `connections`, and `stripe` can be undefined, which would cause an error.

I would suggest either:
a) use a ternary to say `stripe: pnid.connections?.stripe ? { ... } : null`
b) use optional chaining on the properties, i.e `pnid.connections?.stripe?.tier_name`

Option a) would give the `stripe` object the possibility to be null, and option b) would ensure the object would always be present but with possibly null values.